probe/ebpf-check/check_load_ebpf.go

```diff
@@ -0,0 +1,134 @@
+/*
```

Comment:
Why not moving this to tests/flow_ebpf_test.go or probe/ebpf/ebpf_test.go ?

Reply:
you mean probe/ebpf-check/check_load_ebpf.go in probe/ebpf/check_load_ebpf.go ?  because we can't have C and golang (main) for 2 different purpose (don't remember exactly the behavior, but try you will see golang restriction :) )

Otherwise : 
 tests/flow_ebpf_test.go  is equivalent to  tests/flow_test.go